### ISSUE TITLE
Workspaces are sorted in alphabetical order

### ISSUE THREAD

**Issue by aleemont** [morrolinux/i3expo-ng#11]
[10th...19th] workspaces are placed after 1st and before 2nd, [20th...29th] between 2nd and 3rd and so on.
This is not a big problem, but may be confusing with _massive_ usage of workspaces. Any clue about how to fix it?

**Comment by morrolinux** [comment]
Yeah... the thing is I've had to switch to string type to support named workspaces, hence the alphabetical order. 
It could be worked around with a conditional cast to int but maybe there are better ways to fix that, I'll have to think about it :)
